routes.js

```diff
@@ -0,0 +1,102 @@
+var Todo = require('./models/todo')(Todo);
+
+module.exports = (server) => {
+
+    function errorMessage(testName, error) {
+        console.log("There is an error with " + testName + " : " + error);
+    }
+    function objectNotFound(res, id){
```

Comment:
This does indeed run. The function is called within this file (for instance in server.get('/todos/:id') )